roles/nginx/templates/proxy.conf.j2

```diff
@@ -26,6 +26,7 @@ server {
     proxy_set_header  Host $host;
     proxy_set_header  X-Real-IP $remote_addr;
     proxy_set_header  X-Forwarded-For $proxy_add_x_forwarded_for;
+    proxy_set_header  X-Forwarded-Proto $scheme;
```

Comment:
@vtepliuk Not having this initially is why your WP site didn't know to produce `https://` urls.